testing_and_setup/compass/ocean/baroclinic_channel/template_forward.xml

```diff
@@ -1,21 +1,12 @@
 <template>
 	<namelist>
-		<option name="config_ocean_run_mode">'forward'</option>
-		<option name="config_dt">'00:00:30'</option>
-		<option name="config_btr_dt">'00:00:02'</option>
-		<option name="config_run_duration">'0000_00:15:00'</option>
-		<option name="config_block_decomp_file_prefix">'graph.info.part.'</option>
-		<option name="config_pio_num_iotasks">1</option>
-		<option name="config_pio_stride">4</option>
 		<option name="config_write_output_on_startup">.false.</option>
-		<option name="config_time_integrator">'split_explicit'</option>
```

Comment:
Why remove?  This helps make things clearer.

Reply:
Why does this help to make things clearer in this one test case, when we just assume it's the default in nearly all other test cases?

Reply:
Because we will have multiple time stepping options in the future and this is more explicit.  We are already running with RK4 for the hurricane cases, for instance, and the new implicit barotropic solve is in progress, not to mention the spatially variable time stepping work starting in the next week or so.

Reply:
Hmm, okay, I get your point.  I still think this is of negligible importance.  Adding these tags will be quite a chore and one test case fewer likely won't make a difference.  But also won't do any harm.

Reply:
Maybe we just leave them out but make sure as they are changed to something else they are specified.  It is hard for me to foresee what will be clearer.

Reply:
My philosophy here is to have template and config file contain only the flags that change at that level. So the 1km/5km/10km forward templates contains the time step and viscosity change only. 

In addition, even at the configuration level (baroclinic channel), I would prefer not to specify namelist options that are both the default setting and uniform for all cases within that configuration. For example, all baroclinic channel cases use `split_explicit`, the default, so it's not needed in the template. I agree that if you have a set of cases like your hurricanes, some with the default and some with non-default settings, it makes sense to explicitly state the default setting.